### PR TITLE
perfect: supplementary avatar type

### PR DESCRIPTION
**Pull request by cetr**

### CODE REVIEW

**Comment by JohnNiang** [comment]
Thanks for your contribution, @cetr .

The improvment is awesome. Please look at the result:

image

**Review by ruibaby — APPROVED** [review]

**Review by guqing — APPROVED** [review]

File: src/views/system/optiontabs/CommentTab.vue [hunk 1]
```diff
@@ -3,13 +3,66 @@
...
       <a-form-model-item label="评论者头像：">
         <a-select v-model="options.comment_gravatar_default">
-          <a-select-option value="mm">默认</a-select-option>
-          <a-select-option value="identicon">抽象几何图形</a-select-option>
-          <a-select-option value="monsterid">小怪物</a-select-option>
-          <a-select-option value="wavatar">Wavatar</a-select-option>
-          <a-select-option value="retro">复古</a-select-option>
-          <a-select-option value="robohash">机器人</a-select-option>
-          <a-select-option value="blank">不显示头像</a-select-option>
+          <a-select-option value="">
```
Comment (JohnNiang) [inline src/views/system/optiontabs/CommentTab.vue:6]: I'd prefer to provide a JSON to render the option comp.

Comment (cetr) [reply src/views/system/optiontabs/CommentTab.vue:6]: 👌

**Comment by JohnNiang** [comment]
LGTM

**Pull request merged**